torch/quantization/observer.py

```diff
@@ -238,10 +318,54 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         super(MinMaxObserver, self)._load_from_state_dict(state_dict, prefix, local_metadata, False,
                                                           missing_keys, unexpected_keys, error_msgs)
 
+
 class MovingAverageMinMaxObserver(MinMaxObserver):
-    def __init__(self, averaging_constant=0.01, **kwargs):
+    r"""Observer module for computing the quantization parameters based on the
+    moving average of the min and max values.
+
+    This observer computes the quantization parameters based on the moving
+    averages of minimums and maximums of the incoming tensors. The module
+    records the average minimum and maximum of incoming tensors, and uses this
+    statistic to compute the quantization parameters.
+
+    Args:
+        averaging_constant: Averaging constant for min/max.
+        dtype: Quantized data type
+        qscheme: Quantization scheme to be used
+        reduce_range: Reduces the range of the quantized data type by 1 bit
+
+    The moving average min/max is computed as follows
+
+    .. math::
+
+        \begin{array}{ll}
+                x_\text{min} = \begin{cases}
+                    \min(X) & \text{if~}x_\text{min} = \text{None} \\
+                    (1 - c) x_\text{min} + c \min(X) & \text{otherwise}
+                \end{cases}\\
+                x_\text{max} = \begin{cases}
+                    \max(X) & \text{if~}x_\text{max} = \text{None} \\
+                    (1 - c) x_\text{max} + c \max(X) & \text{otherwise}
+                \end{cases}\\
+        \end{array}
```

Comment:
![image](https://user-images.githubusercontent.com/4216323/66872743-8a212080-ef5b-11e9-99b3-10646f71efa9.png)

Reply:
probably say `c` is `averaging_constant` here as well?

Reply:
> probably say `c` is `averaging_constant` here as well?

You mean in the equation? I say it in the text right after. So it will be rendered as equation followed by the variable description. I can add the description in the equation tho